scss/components/_search-bar.scss

```diff
@@ -42,7 +42,8 @@
   .combo__input {
     border-radius: 4px 0 0 4px;
     float: left;
-    width: calc(100% - 5.6rem);
+    $search-button-width: u(5.6rem);
```

Comment:
I like assigning this to a variable (in addition to solving the bug). What about also using it when defining the width of the button? Also, can we move the variable assignment to the top of the list?